<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Applied and Environmental Microbiology"><meta name="citation_title" content="Recovery and Phylogenetic Analysis of nifH Sequences from Diazotrophic Bacteria Associated with Dead Aboveground Biomass of Spartina alterniflora"><meta name="citation_authors" content="Charles R. Lovell, Michael J. Friez, John W. Longshore, and  Christopher E. Bagwell"><meta name="citation_date" content="2001 November"><meta name="citation_issue" content="11"><meta name="citation_volume" content="67"><meta name="citation_firstpage" content="5308"><meta name="citation_doi" content="10.1128/AEM.67.11.5308-5314.2001"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=93305&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=93305&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=93305"><meta name="citation_pmid" content="11679360"><meta name="DC.Title" content="Recovery and Phylogenetic Analysis of nifH Sequences from Diazotrophic Bacteria Associated with Dead Aboveground Biomass of Spartina alterniflora"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="American Society for Microbiology (ASM)"><meta name="DC.Contributor" content="Charles R. Lovell"><meta name="DC.Contributor" content="Michael J. Friez"><meta name="DC.Contributor" content="John W. Longshore"><meta name="DC.Contributor" content="Christopher E. Bagwell"><meta name="DC.Date" content="2001 November"><meta name="DC.Identifier" content="10.1128/AEM.67.11.5308-5314.2001"><meta name="DC.Language" content="en"><link rel="stylesheet" href="corehtml/pmc/css/pmcstatic.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbase1.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbars-slateblue.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbody9.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcrefs4.css" type="text/css"><script type="text/javascript" src="corehtml/pmc/js/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/css/tileshop_pmc.1/tileshop_pmc.1.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/utils.1.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/tileshop_pmc.1/tileshop_pmc.1.js"></script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corecgi/tileshop/tileshop_data_db.1.js"></script><title>Recovery and Phylogenetic Analysis of nifH Sequences from Diazotrophic Bacteria Associated with Dead Aboveground Biomass of Spartina alterniflora</title></head><body class="pmc-body"><a id="top" name="top"></a><table cellpadding="0" cellspacing="0" width="100%" border="0"><tr valign="top"><td colspan="3"><table border="0" cellspacing="0" cellpadding="0" style="border-collapse: collapse;"><tr><td width="145" valign="top"><img src="corehtml/pmc/pmcgifs/pmc3_logo_v5.gif" border="0" usemap="#pmclogo" alt="pmc logo image"><map name="pmclogo"><area shape="rect" alt="Journal List" coords="65,52,137,67" href="fprender.fcgi"><area shape="rect" alt="Search" coords="7,52,52,67" href="redirect3.cgi?&amp;&amp;auth=0VsrPEOvbIOBVwKGUr4p5LUHeI-8QNpJkOwRYZQT6&amp;reftype=other&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="./"></map></td><td height="75" valign="top" style="padding-left: 4px;"><div><img align="top" border="0" src="corehtml/pmc/pmcgifs/logo-aem.gif" alt="Logo of aem" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" alt="Formerly Appl Microbiol" coords="193,34,500,48" href="tocrender.fcgi?issn=0003-6919&amp;action=archive"><area shape="rect" alt="Appl Environ Microbiol Subscriptions" coords="0,64,190,75" href="redirect3.cgi?&amp;&amp;auth=0p9TsdtS6xGsZ-8jBvC9CDV5UkiD9q2RJGp5wO5OR&amp;reftype=publisher&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://aem.asm.org/subscriptions/" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="DEFAULT" alt="Appl Environ Microbiol Web Site" coords="0,0,500,75" href="redirect3.cgi?&amp;&amp;auth=0z1Sx-flrsyLpHQ8kE8jWJnZ-1z23E2bVhNr_dAe-&amp;reftype=publisher&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://aem.asm.org" target="pmc_ext" onclick="focuswin('pmc_ext')"></map></div></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="3" width="100%" border="0"><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-gray"><a href="fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?journal=83&amp;action=archive">Appl Environ Microbiol</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?iid=2543">v.67(11); Nov 2001</a></div></td></tr><tr><td width="145" valign="top" class="sidebar-cell"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="articlerender.fcgi?artid=93305&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="picrender.fcgi?artid=93305&amp;blobtype=pdf" class="sidefm-pmclink">PDF (510K)</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?iid=2543" class="sidefm-pmclink">Contents</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?journal=83&amp;action=archive" class="sidefm-pmclink">Archive</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?journal=83&amp;action=homepage" class="sidefm-pmclink">Journal Homepage</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=0v96AKqg-Dg91EFKYIdFzvGM_ryLRCnKGJyTlEQ6A&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11679360">PubMed record</option><option selected value="redirect3.cgi?&amp;&amp;auth=0J5x1W7PVB6t9kwB60JrU4Jp04aZtzEM6goT_PAsq&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=11679360">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0tgm4TdW51uL-GA_1_r1FlqE2Mxk7shF0YlCPEQjT&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=11679360">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=00qVDH2TRYWOcwgEBEVVEuEoxvBdtSuXRpcEzWkx1&amp;reftype=Nucleotide&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Nucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=93305&amp;dopt=pmc_nuccore">Nucleotide</option><option value="redirect3.cgi?&amp;&amp;auth=0R9qqW3NLiQamtyiZi1P6_oO1rjrX7vUFBpwg0575&amp;reftype=Substance&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|PubChem%20Substance&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=93305&amp;dopt=pmc_pcsubstance">Substance</option><option value="redirect3.cgi?&amp;&amp;auth=0LIRN1428MKAdKcfFbJG9U-h8ItooR8-UYsEACAMV&amp;reftype=Protein&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Protein&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=93305&amp;dopt=pmc_protein">Protein</option><option value="redirect3.cgi?&amp;&amp;auth=0PKok6KEr4GGM95ewYGLWHTb_hItXtcaqSQHskQ-R&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=93305&amp;dopt=pmc_taxonomy">Taxonomy</option><option value="redirect3.cgi?&amp;&amp;auth=0-Ay2f39yR8Y4lxdjkKIm5k9XXLzMbi5LGpCgz5nh&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?Db=taxonomy&amp;DbFrom=pmc&amp;cmd=Link&amp;LinkName=pmc_taxonomy&amp;doptcmdl=TxTree&amp;IdsFromResult=93305">Taxonomy tree</option></select><br><input type="image" src="corehtml/pmc/pmcgifs/go.gif" alt="Go" class="sidefm-pmart"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0xIhcOxD9pYVbIgNgX4Qu_DP71aZZC04a4yEJhAbd&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Lovell+CR%5Bauth%5D" class="sidefm-pmclink"> Lovell, C.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0-5bz69RiTD_pO8vULZ5Bwt1kPQQkkJYhrYCkLQol&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Friez+MJ%5Bauth%5D" class="sidefm-pmclink"> Friez, M.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0z1yTK-qEguzB1jkYkJHjhzYyndgPSDuN4uCwFQ3Q&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Longshore+JW%5Bauth%5D" class="sidefm-pmclink"> Longshore, J.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0TE_qXjbZgiYgdw7221pntlUXjXGxnSpyYYqx5ALy&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Bagwell+CE%5Bauth%5D" class="sidefm-pmclink"> Bagwell, C.</a></div></div></div></td><td rowspan="100" width="20px" height="300" background="corehtml/pmc/pmcgifs/wm-aem.gif"><div style="width:20px"></div></td><td valign="top" class="content-cell"><div class="front-matter-section"><table cellspacing="0" cellpadding="0" width="100%"><tr style="vertical-align: top"><td><div class="fm-citation"><div><span class="citation-abbreviation">Appl Environ Microbiol. </span><span class="citation-publication-date">2001 November; </span><span class="citation-volume">67</span><span class="citation-issue">(11)</span><span class="citation-flpages">: 5308–5314. </span></div><div><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1128/AEM.67.11.5308-5314.2001.</span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC93305</div></td></tr></table><div class="fm-copyright"><a class="int-reflink" href="about/copyright.html">Copyright</a>  &#x000a9; 2001, American Society for Microbiology</div><div class="fm-title">Recovery and Phylogenetic Analysis of <em>nifH</em> Sequences from Diazotrophic Bacteria Associated with Dead Aboveground Biomass of <em>Spartina alterniflora</em></div><div class="fm-author contrib-group">Charles R. Lovell,<sup>1</sup><sup>&#x0002a;</sup> Michael J. Friez,<sup>2</sup> John W. Longshore,<sup>2</sup> and  Christopher E. Bagwell<sup>1</sup><sup>&#x02020;</sup><div class="fm-affl"> Department of Biological Sciences, University of South Carolina, Columbia, South Carolina 29208,<sup>1</sup> and  Molecular Diagnostics Laboratory, Greenwood Genetic Center, Greenwood, South Carolina 29646<sup>2</sup></div></div><div class="fm-footnote"></div><div class="fm-footnote" id="FN150"><sup>&#x0002a;</sup>Corresponding author. Mailing address: Department of Biological Sciences, University of South Carolina, Columbia, SC 29208. Phone: (803) 777-7036. Fax: (803) 777-4002. E-mail: <span class="e_id674449">lovell/at/biol.sc.edu</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id674449", '<a class="ext-reflink" href="' + reverseAndReplaceString('ude.cs.loib/ta/llevol:otliam', '/at/', '@') + '">' + reverseAndReplaceString('ude.cs.loib/ta/llevol', '/at/','@') + '</a>')}catch(e){}
                                //--></script>.</div><div class="fm-footnote" id="FN151"><sup>&#x02020;</sup>Present address: Environmental Sciences Division, Oak Ridge National Laboratory, Oak Ridge, TN 37831.</div><div class="fm-pubdate">Received June 4, 2001; Accepted August 21, 2001.</div><div class="links-box"><div class="fm-footnote"><img src="corehtml/pmc/pmcgifs/rt-arrow.gif" alt="Small right arrow pointing to:" style="vertical-align: middle;"> This article has been <a class="int-reflink" href="tocrender.fcgi?artid=93305&amp;action=cited">cited by</a> other articles in PMC.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Abstract</a></div><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id671778" class="sidefm-pmclink" style="text-transform: uppercase;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id701812">Abstract</div><div class="section-content"><!--article-meta-->DNA was extracted from dry standing dead <em>Spartina alterniflora</em> stalks as well as dry <em>Spartina</em> wrack from the North Inlet (South Carolina) and Sapelo Island (Georgia) salt marshes. Partial <em>nifH</em> sequences were PCR amplified, the products were separated by denaturing gradient gel electrophoresis (DGGE), and the prominent DGGE bands were sequenced. Most sequences (109 of 121) clustered with those from &#x003b1;-<em>Proteobacteria</em>, and 4 were very similar (&#x0003e;99&#x00025;) to that of <em>Azospirillum brasilense</em>. Seven sequences clustered with those from known &#x003b3;-<em>Proteobacteria</em> and five with those from known anaerobic diazotrophs. The diazotroph assemblages associated with dead <em>Spartina</em> biomass in these two salt marshes were very similar, and relatively few major lineages were represented.</div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id701812" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id671778" class="sidefm-pmclink" style="text-transform: uppercase;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id631190"> </div><div class="section-content">Low elevations of salt marshes along the Atlantic and northern Gulf of Mexico coasts of temperate North America are characterized by extensive monoculture stands of the smooth cordgrass <em>Spartina alterniflora</em> (<a href="#B35" class="cite-reflink">35</a>). <em>Spartina</em> marshes support high rates of macrophyte primary production and microbially mediated nutrient cycling. Numerous studies indicate that primary production (<a href="#B16" class="cite-reflink">16</a>, <a href="#B37" class="cite-reflink">37</a>) and decomposition (<a href="#B15" class="cite-reflink">15</a>, <a href="#B19" class="cite-reflink">19</a>, <a href="#B38" class="cite-reflink">38</a>) in <em>Spartina</em> marshes are nitrogen limited. In these systems, diazotrophy (N<sub>2</sub> fixation) is an important source of &#x0201c;new&#x0201d; nitrogen (<a href="#B8" class="cite-reflink">8</a>, <a href="#B23" class="cite-reflink">23</a>, <a href="#B40" class="cite-reflink">40</a>).<p>A significant but often overlooked focus of diazotrophy in salt marshes is dead aboveground <em>Spartina</em> biomass, particularly standing dead biomass (<a href="#B19" class="cite-reflink">19</a>, <a href="#B33" class="cite-reflink">33</a>). There are quite large amounts of standing dead biomass at all times of the year, with ratios of dead to live aboveground biomass exceeding 1:1 during the winter and spring months (<a href="#B7" class="cite-reflink">7</a>, <a href="#B31" class="cite-reflink">31</a>). Standing dead biomass is partially mineralized and frequently very dry (<a href="#B17" class="cite-reflink">17</a>) but supports substantial microbial activity (<a href="#B17" class="cite-reflink">17</a>&#x02013;<a href="#B19" class="cite-reflink">19</a>), which is greatly stimulated when the material becomes wet through tidal action or precipitation (<a href="#B17" class="cite-reflink">17</a>). Rates of diazotrophy in moist standing dead <em>Spartina</em> are among the highest reported for decomposing plant materials (see Table 4 in reference <a href="#B19" class="cite-reflink">19</a>).</p><p>Relatively little attention has been given to the microorganisms involved in decomposition of and diazotrophy in dead aboveground <em>Spartina</em> biomass. Much of the microbial biomass in standing dead materials consists of fungal hyphae (<a href="#B18" class="cite-reflink">18</a>). Cyanobacteria are present but occur chiefly in clay-rich surface films (<a href="#B18" class="cite-reflink">18</a>), while diazotrophy occurs primarily on and within the decaying biomass itself (<a href="#B19" class="cite-reflink">19</a>). It seems that the predominant diazotrophs in this material are heterotrophic bacteria, but the types of organisms present have not been determined.</p><p>Recent applications of molecular biological methods have greatly facilitated the study of natural bacterial communities and functionally significant taxa within them (<a href="#B9" class="cite-reflink">9</a>, <a href="#B34" class="cite-reflink">34</a>, <a href="#B39" class="cite-reflink">39</a>, <a href="#B42" class="cite-reflink">42</a>). In particular, PCR amplification has been used to recover segments of <em>nifH</em>, the structural gene encoding the nitrogenase iron protein, from various types of environmental samples, including marine and freshwater plankton (<a href="#B1" class="cite-reflink">1</a>, <a href="#B3" class="cite-reflink">3</a>, <a href="#B44" class="cite-reflink">44</a>), termite hindguts (<a href="#B11" class="cite-reflink">11</a>, <a href="#B20" class="cite-reflink">20</a>), microbial mats and aggregates (<a href="#B21" class="cite-reflink">21</a>, <a href="#B22" class="cite-reflink">22</a>, <a href="#B43" class="cite-reflink">43</a>), terrestrial soils (<a href="#B28" class="cite-reflink">28</a>, <a href="#B29" class="cite-reflink">29</a>, <a href="#B32" class="cite-reflink">32</a>, <a href="#B41" class="cite-reflink">41</a>), the rhizoplanes of rice (<em>Oryza sativa</em>) (<a href="#B36" class="cite-reflink">36</a>) and of shoal grass (<em>Halodule wrightii</em>) (<a href="#B10" class="cite-reflink">10</a>), and the rhizosphere of <em>Spartina</em> (<a href="#B14" class="cite-reflink">14</a>). PCR amplification of <em>nifH</em> sequences followed by their separation through denaturing gradient gel electrophoresis (DGGE) has been used to examine the complexity and stability of the diazotroph assemblage found in the <em>Spartina</em> rhizosphere (<a href="#B25" class="cite-reflink">25</a>&#x02013;<a href="#B27" class="cite-reflink">27</a>), and sequence analysis of the DGGE bands has been used to determine phylogenetic relationships of the diazotrophic organisms represented (<a href="#B14" class="cite-reflink">14</a>). While such methods have certain inherent limitations and biases (<a href="#B25" class="cite-reflink">25</a>, <a href="#B42" class="cite-reflink">42</a>), they provide an efficient means to profile the diazotrophs associated with dead aboveground <em>Spartina</em> biomass and to determine the phylogenetic affiliations of these organisms.</p><p>In this study we determined the types of diazotrophic heterotrophic bacteria present in standing dead and loose, recently deposited (wrack) <em>Spartina</em> biomass, as defined by recoverable partial <em>nifH</em> sequences resolved by DGGE. Our primary objectives were to assess the diversity of these assemblages and to identify the major phylogenetic groups of organisms that are capable of contributing to N<sub>2</sub> fixation in dead aboveground <em>Spartina</em> biomass.</p><p><span class="head3">Sampling sites. </span> Samples of standing dead <em>Spartina</em> stalks and <em>Spartina</em> wrack were collected from the short-form <em>Spartina</em> zones in two different salt marsh systems. The Crab Haul Creek Basin site in the North Inlet estuary, near Georgetown, S.C. (79&#x000b0;12&#x02032;W, 33&#x000b0;20&#x02032;N), was located in the intertidal zone approximately 50 m from the nearest tidal creek and was sampled on 31 August 2000. The Doboy Sound site on Sapelo Island, Ga. (31&#x000b0;23&#x02032;N, 81&#x000b0;17&#x02032;W), was located in the intertidal zone near Doboy Sound (Georgia Coastal Ecosystems Long Term Ecological Research Site 6) and was sampled on 1 August 2000. The upper, approximately 10-cm lengths of dry standing dead stalks were collected. Dry wrack, which consisted of loose stalks (litter) recently deposited on the sediment surface, was collected from deposits lying near the sampling locations for standing dead stalks. Standing dead stalks and wrack were transferred to sterile Whirl-Pak bags and stored at &#x02212;70&#x000b0;C pending DNA extraction.</p><p><span class="head3">DNA extraction. </span> Standing dead and wrack stalks were broken up into 2-cm fragments. DNA was extracted from the samples using a direct lysis procedure described previously (<a href="#B13" class="cite-reflink">13</a>, <a href="#B25" class="cite-reflink">25</a>). DNA extracts were further purified and concentrated using the Wizard DNA clean-up system following the manufacturer's instructions (Promega, Madison, Wis.). DNA quality and quantity were assessed by agarose gel electrophoresis and fluorometry, respectively.</p><p><span class="head3">PCR amplification of <em>nifH</em>. </span> PCR was performed using <em>Taq</em> DNA polymerase (Qiagen, Valencia, Calif.) in a reaction mixture containing (25-&#x003bc;l reaction volume) 25 ng of template DNA, 1.5 mM MgCl<sub>2</sub>, 2 &#x003bc;M deoxynucleoside triphosphate (dNTP) mixture, 0.5 pmol of each primer, and 10 &#x003bc;g of bovine serum albumin. The <em>nifH</em> primers used were those of Piceno et al. (<a href="#B25" class="cite-reflink">25</a>) and are specific for heterotrophic diazotrophs. These primers were designed to have low degeneracy, which is needed for DGGE applications, and are not expected to amplify <em>nifH</em> sequences from cyanobacteria, <em>Frankia</em> spp., and methanogens. Primer design and testing have been described previously (<a href="#B25" class="cite-reflink">25</a>). Amplification was initiated by a denaturation step at 94&#x000b0;C for 2 min and proceeded in two phases: (i) a 20-cycle touchdown program (94&#x000b0;C for 45 s, 58&#x000b0;C for 30 s, decreasing 0.5&#x000b0;C/cycle, and 70&#x000b0;C for 30 s), and (ii) 20 cycles of a standard amplification program at a 48&#x000b0;C annealing temperature for 30 s. A final extension step at 70&#x000b0;C for 2 min was used. Multiple individual reactions were performed for each sample. PCR products were pooled (200-&#x003bc;l final volume per sample) and stored as alcohol precipitates at &#x02212;20&#x000b0;C. Prior to DGGE, amplimers were recovered by centrifugation and dissolved in 15 &#x003bc;l of TE (10 mM Tris-HCl &#x0005b;pH 8.0&#x0005d;, 1 mM EDTA).</p><p><span class="head3">DGGE. </span> <em>nifH</em> amplimers were electrophoresed on denaturing gradient gels (1-mm thick, 6.5&#x00025; polyacrylamide, 78 to 89&#x00025; denaturant, where 100&#x00025; denaturant contains 7 M urea and 40&#x00025; formamide) at 48&#x000b0;C for 1,900 V<img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />&#x000b7;<img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />h using the Bio-Rad (Hercules, Calif.) DCode universal mutation detection system. Gels were stained for 30 min in TE with SYBR Gold (Molecular Probes, Eugene, Oreg.) and documented using the Alpha Imager 2000 (Alpha Innotech Corp., San Leandro, Calif.). Gel plugs were collected from all well-resolved bands in the profiles using wide-bore micropipette tips and stored in 50 &#x003bc;l of distilled water at &#x02212;20&#x000b0;C. Gel bands were designated homoduplexes or heteroduplexes following previously described methods (<a href="#B25" class="cite-reflink">25</a>).</p><p><span class="head3">Amplimer cloning and identification of different cloned amplimer sequences. </span> Amplimers from DGGE gel plugs were recovered by reamplification and cloned as described previously (<a href="#B14" class="cite-reflink">14</a>). Recombinant colonies were maintained on Luria-Bertani agar plates containing 100 &#x003bc;g of ampicillin ml<sup>&#x02212;1</sup>. Clones were screened for appropriately sized insert by amplification using primers specific for the SP6 and T7 RNA polymerase binding sites (<a href="#B14" class="cite-reflink">14</a>). Restriction fragment length polymorphism analysis was employed to assess gel band amplimer composition (i.e., homogeneous or heterogeneous) and to identify different clones for sequencing (<a href="#B14" class="cite-reflink">14</a>).</p><p><span class="head3">DNA sequencing and analysis. </span> Recombinant plasmids were purified from selected clones by using the Qiagen Plasmid Mini Kit. Plasmid concentrations were determined fluorometrically. Sequencing reactions used T7 and Sp6 primers and ABI (Applied Biosystems, Foster City, Calif.) BigDye version 2.0 chemistry. Sequences were determined using an ABI 3100 genetic analyzer. For phylogenetic reconstructions, <em>nifH</em> sequences from numerous different known diazotrophs and from various environmental sources were selected using the Blast search feature of the NCBI GenBank database (<a href="#B2" class="cite-reflink">2</a>). Nucleotide sequences were translated, and the inferred amino acid sequences were aligned (<a href="#B5" class="cite-reflink">5</a>) and checked by hand for proper alignment of conserved marker residues (<a href="#B14" class="cite-reflink">14</a>). Neighbor-joining phylogenies (<a href="#B30" class="cite-reflink">30</a>) were constructed in MEGA version 2.0 (<a href="#B12" class="cite-reflink">12</a>) using percent dissimilarity distances and pairwise deletion of gaps and missing data. The use of alternative amino acid distance measures (e.g., Poisson and gamma correction for multiple substitutions) or tree construction methods (neighbor joining or Unweighted Pair Group Method Using Arithmetic Averages) had no significant effect on the resulting dendrogram topology (data not shown). NifH amino acid sequences from <em>Methanobacterium thermoautotrophicum</em> (accession no. AE00916) and <em>Methanosarcina barkeri</em> (<span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0DY9fc4TrUN0eIXS8daoaNfBisSWOTqAYSNbuW5S6&amp;reftype=extlink-entrez-nucleotide&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CNucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=nucleotide&amp;dopt=GenBank&amp;list_uids=1772570" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">X56072</a></span>) were used as outgroup taxa. Bootstrapping (<a href="#B6" class="cite-reflink">6</a>) was used to estimate the reliability of phylogenetic reconstructions (500 replicates).<p>All dead aboveground <em>Spartina</em> biomass samples yielded strong amplification products, although the North Inlet samples produced substantially stronger amplification than the Sapelo Island samples (Fig. <a href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">1</a>). DGGE yielded 16 well-resolved <em>nifH</em> amplimer bands from North Inlet standing dead biomass and 9 bands from wrack. Due to somewhat lower yields of amplification products from the Sapelo Island samples, only five robust bands were recovered from the standing dead material and four bands from wrack. All of these bands were successfully sampled, and the amplimers were cloned and screened for unique sequences. An artifact band, which was described previously (<a href="#B25" class="cite-reflink">25</a>), was also observed in each sample lane. A total of 54 different partial <em>nifH</em> sequences were recovered from standing dead <em>Spartina</em> and 24 from <em>Spartina</em> wrack collected at North Inlet. Twenty-one different sequences were recovered from the standing dead material and 20 from the wrack from Sapelo Island.  </p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F1" name="F1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=93305&amp;blobname=am1110925001.gif" border="1" class="icon-reflink" alt="FIG. 1" title="FIG. 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>FIG. 1</strong></a><div class="figure-table-caption-in-article"><span>Denaturing gradient gel images showing <em>nifH</em> amplimers from dead aboveground <em>Spartina</em> biomass. PCR and DGGE conditions are described in Materials and Methods. Lanes: A, North Inlet standing dead <em>Spartina</em>; B, North Inlet <em>Spartina</em> wrack; C, Sapelo Island</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>The partial <em>nifH</em> sequences were initially translated and examined for key, highly conserved amino acid residues that are important in nitrogenase iron protein structure and function (<a href="#B4" class="cite-reflink">4</a>, <a href="#B24" class="cite-reflink">24</a>). Within the segments analyzed, 11 amino acids, including (<em>Klebsiella pneumoniae</em> &#x0005b;<span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0jEtPxeDBuB0cFb4MojA4WE5AqxxHcjM0R3icdkc9&amp;reftype=extlink-entrez-nucleotide&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CNucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=nucleotide&amp;dopt=GenBank&amp;list_uids=149259" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">J01740</a></span>&#x0005d; numbering) Lys15 and Ser16 (within the MgATP binding domain), Arg100 (the ADP-ribosylation site), Asp125 (possibly involved in protein conformation changes), Asp129 (involved in ATP hydrolysis), Arg140 and Lys143 (contribute to salt bridge formation), and four conserved Cys residues (numbers 38, 85, 97, and 132, two of which coordinate the Fe<sub>4</sub>S<sub>4</sub> cluster), were used as markers for determining sequence accuracy. Four sequences had one substitution each: NIS2-1 and SIW2-6, Cys85 replaced by Arg85; NIS3-2, Ser16 replaced by Trp16; and NIW8-1, Cys97 replaced by Arg97. All sequences were used in subsequent analyses.</p><p>The sequences from dead aboveground <em>Spartina</em> biomass and their closest affiliations with known diazotrophs on the basis of sequence similarity are listed in Table <a href="articlerender.fcgi?artid=93305&amp;rendertype=table&amp;id=T1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">1</a>. As has been reported in numerous previous studies of environmental <em>nifH</em> sequences (<a href="#B1" class="cite-reflink">1</a>, <a href="#B3" class="cite-reflink">3</a>, <a href="#B14" class="cite-reflink">14</a>, <a href="#B29" class="cite-reflink">29</a>, <a href="#B36" class="cite-reflink">36</a>, <a href="#B41" class="cite-reflink">41</a>, <a href="#B43" class="cite-reflink">43</a>, <a href="#B44" class="cite-reflink">44</a>), these sequences fell into three major clusters. The overwhelming majority of dead <em>Spartina nifH</em> sequences were affiliated with a cluster defined by sequences from &#x003b1;-<em>Proteobacteria</em> and well supported by bootstrapping (Fig. <a href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a>). Forty-six of 54 sequences (85&#x00025;) from the North Inlet standing dead sample, all 24 of the sequences from North Inlet wrack (100&#x00025;), 19 of 21 sequences (90&#x00025;) from the Sapelo Island standing dead sample, and 17 of 20 sequences (85&#x00025;) from Sapelo Island wrack were from presumptive &#x003b1;-<em>Proteobacteria</em>. These sequences were further subdivided into a number of smaller clusters, some of which contained <em>nifH</em> sequences from known diazotrophs.    </p><div><div class="p"><strong>TABLE 1</strong></div><div class="p"> <div class="p"><p>Similarities of dead aboveground <em>Spartina</em> biomass NifH amino acid sequences to the most similar sequences from known diazotrophic bacterial species<sup>a</sup> </p></div></div></div><div><table cellspacing="3" cellpadding="2" frame="hsides" rules="groups" class="rendered"><thead><tr><th>Dead <em>Spartina</em> sequence(s)</th><th>Organism(s) with most similar sequence(s)</th><th>&#x00025; Similarity </th></tr></thead><tbody align="center"><tr><td align="left">Presumptive &#x003b1;-<em>Proteobacteria</em></td><td></td><td></td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS2-1</td><td align="left"><em>Gluconacetobacter diazotrophicus</em></td><td>95.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS2-2, NIS2-3, NIS4-2, NIS4-4, NIS5-1, NIS5-2, NIS6-1, NIS6-2, NIS7-1, NIS8-1, NIS9-2, NIS10-2, NIS12-3, NIS13-1, NIS13-2, NIW4-1, NIW6-2</td><td align="left"><em>Rhodobacter sphaeroides</em></td><td>94.4 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS3-1, NIS6-2, NIS7-3</td><td align="left"><em>Azospirillum brasilense</em></td><td>94.4 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS3-2, NIS9-1, NIS10-3, NIS14-1, NIS15-2, NIS15-4, NIS16-3</td><td align="left"><em>Bradyrhizobium japonicum</em></td><td>97.6 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS4-1</td><td align="left"><em>Azospirillum brasilense, Gluconacetobacter diazotrophicus</em></td><td>92.8 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS4-3</td><td align="left"><em>Rhodobacter sphaeroides</em></td><td>95.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS7-2, NIW2-3, NIW4-2, NIW5-1, NIW8-2, NIW8-3, NIW9-4</td><td align="left"><em>Rhizobium</em> sp.</td><td>94.4 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS8-2</td><td align="left"><em>Rhodobacter sphaeroides</em></td><td>92.8 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS8-3, NIW3-1</td><td align="left"><em>Rhodobacter sphaeroides</em></td><td>96.0 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS10-1, NIS13-3</td><td align="left"><em>Rhodobacter sphaeroides</em></td><td>96.8 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS10-4</td><td align="left"><em>Azospirillum brasilense</em></td><td>88.0 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS11-1</td><td align="left"><em>Rhodobacter capsulatus, Rhodobacter sphaeroides</em></td><td>97.6 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS11-2</td><td align="left"><em>Azospirillum brasilense</em></td><td>92.0 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS11-3</td><td align="left"><em>Rhodobacter sphaeroides</em></td><td>98.4 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS12-1</td><td align="left"><em>Bradyrhizobium japonicum, Rhizobium leguminosarum</em> biovar phaseoli</td><td>95.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS12-2</td><td align="left"><em>Rhodobacter sphaeroides</em></td><td>93.6 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS13-4</td><td align="left"><em>Herbaspirillum seropedicae, Rhizobium</em> sp.</td><td>92.0 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS14-3, NIS14-4, NIS15-3, NIS16-4</td><td align="left"><em>Rhodobacter sphaeroides</em></td><td>97.6 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS15-1</td><td align="left"><em>Azospirillum brasilense</em></td><td>91.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS16-2, NIW8-1, NIW9-1</td><td align="left"><em>Rhizobium</em> sp.</td><td>92.8 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIW2-2, NIW6-3, SIS5-1</td><td align="left"><em>Herbaspirillum seropedicae, Rhizobium</em> sp.</td><td>92.8 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIW2-4</td><td align="left"><em>Azospirillum brasilense, Gluconacetobacter diazotrophicus</em></td><td>96.8 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIW3-2, NIW6-1</td><td align="left"><em>Herbaspirillum seropedicae, Rhizobium</em> sp.</td><td>92.0 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIW3-3, NIW4-3, NIW5-2, NIW6-3, NIW7-1, NIW9-2, NIW9-3</td><td align="left"><em>Rhizobium</em> sp.</td><td>93.6 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIW4-4</td><td align="left"><em>Azospirillum brasilense</em></td><td>98.4 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIW5-3</td><td align="left"><em>Azospirillum brasilense, Gluconacetobacter diazotrophicus</em></td><td>91.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS1-1, SIS4-4</td><td align="left"><em>Azospirillum brasilense, Gluconacetobacter diazotrophicus</em></td><td>94.4 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS2-1, SIS4-2</td><td align="left"><em>Azospirillum brasilense</em></td><td>100.0 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS2-2, SIW1-2</td><td align="left"><em>Gluconacetobacter diazotrophicus</em></td><td>94.4 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS2-3</td><td align="left"><em>Gluconacetobacter diazotrophicus</em></td><td>92.8 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS2-5</td><td align="left"><em>Azospirillum brasilense</em></td><td>92.8 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS2-6</td><td align="left"><em>Gluconacetobacter diazotrophicus</em></td><td>86.4 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS3-1</td><td align="left"><em>Azospirillum brasilense</em></td><td>95.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS3-2, SIW1-7</td><td align="left"><em>Gluconacetobacter diazotrophicus</em></td><td>97.6 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS3-3, SIS3-6</td><td align="left"><em>Azospirillum brasilense</em></td><td>93.6 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS3-4</td><td align="left"><em>Azospirillum brasilense, Bradyrhizobium japonicum, Gluconacetobacter diazotrophicus, Rhodobacter sphaeroides</em></td><td>93.6 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS3-5</td><td align="left"><em>Gluconacetobacter diazotrophicus, Rhizobium</em> sp.</td><td>83.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS3-7</td><td align="left"><em>Azospirillum brasilense, Gluconacetobacter diazotrophicus</em></td><td>92.0 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS3-8, SIW2-3, SIW2-5</td><td align="left"><em>Gluconacetobacter diazotrophicus</em></td><td>93.6 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS4-1, SIW2-6</td><td align="left"><em>Herbaspirillum seropedicae, Rhizobium</em> sp.</td><td>93.6 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS4-3</td><td align="left"><em>Azospirillum brasilense</em></td><td>99.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIW1-5</td><td align="left"><em>Azospirillum brasilense, Bradyrhizobium japonicum, Gluconacetobacter diazotrophicus, Rhizobium leguminosarum</em> biovar phaseoli</td><td>94.4 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIW1-6</td><td align="left"><em>Azospirillum brasilense, Gluconacetobacter diazotrophicus</em></td><td>93.6 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIW2-1</td><td align="left"><em>Rhodobacter sphaeroides</em></td><td>96.0 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIW2-2</td><td align="left"><em>Azospirillum brasilense</em></td><td>96.0 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIW2-4, SIW4-2</td><td align="left"><em>Herbaspirillum seropedicae</em></td><td>92.8 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIW3-1</td><td align="left"><em>Gluconacetobacter diazotrophicus</em></td><td>96.0 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIW4-1</td><td align="left"><em>Gluconacetobacter diazotrophicus</em></td><td>91.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIW4-3</td><td align="left"><em>Rhizobium</em> sp.</td><td>89.6 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SW4-4</td><td align="left"><em>Azospirillum brasilense</em></td><td>99.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SW4-5</td><td align="left"><em>Azospirillum brasilense</em></td><td>97.6  </td></tr><tr><td align="left">Presumptive &#x003b3;-<em>Proteobacteria</em></td><td align="left"></td><td></td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS1-2</td><td align="left"><em>Azomonas agilis, Azotobacter chroococcum</em></td><td>95.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS1-3</td><td align="left"><em>Azomonas agilis, Azotobacter chroococcum</em></td><td>99.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS1-4</td><td align="left"><em>Pseudomonas stutzeri</em></td><td>98.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS11-4</td><td align="left"><em>Azotobacter chroococcum, Azotobacter vinelandii</em></td><td>84.9 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS2-4</td><td align="left"><em>Vibrio diazotrophicus</em></td><td>86.1 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIS2-7</td><td align="left"><em>Vibrio diazotrophicus</em></td><td>84.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIW1-4</td><td align="left"><em>Azomonas agilis</em></td><td>96.8 </td></tr><tr><td align="left">Presumptive anaerobes</td><td align="left"></td><td></td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS1-1</td><td align="left"><em>Desulfovibrio salexigens</em></td><td>84.2 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS3-3</td><td align="left"><em>Desulfonema limicola</em></td><td>84.8 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS14-2</td><td align="left"><em>Desulfonema limicola</em></td><td>82.8 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />NIS16-1</td><td align="left"><em>Desulfovibrio salexigens</em></td><td>88.1 </td></tr><tr><td align="left"><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />SIW1-1</td><td align="left"><em>Desulfovibrio salexigens</em></td><td>85.1</td></tr></tbody></table></div><div class="section-content"><div class="fm-footnote" id="TF1-a"><sup>a</sup>Percent similarities are from the distance matrix constructed in MEGA version 2.0. <em>Spartina</em> sequences are listed by the DGGE gel bands from which they were recovered (see Materials and Methods) and grouped into the major sequence clusters shown in Fig. <a href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a> to <a href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F4" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">4</a>. Note that these clusters are named for the predominant types of organisms that they contain, but some sequences from other phylogenetic groupings can occur.</div></div><div style="clear:both;"></div><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F2" name="F2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=93305&amp;blobname=am1110925002.gif" border="1" class="icon-reflink" alt="FIG. 2" title="FIG. 2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>FIG. 2</strong></a><div class="figure-table-caption-in-article"><span>Phylogenetic analysis of dead aboveground <em>Spartina</em> biomass NifH amino acid sequences from presumptive &#x003b1;-<em>Proteobacteria</em>, from various known &#x003b1;-<em>Proteobacteria</em>, and from selected unknown, presumptive &#x003b1;-<em>Proteobacteria</em> from other sources.</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>Several sequences from North Inlet standing dead <em>Spartina</em> biomass were closely affiliated with purple nonsulfur bacteria, and seven had substantial similarity (&#x02265;96&#x00025;) to the <em>Rhodobacter sphaeroides</em> NifH sequence (Table <a href="articlerender.fcgi?artid=93305&amp;rendertype=table&amp;id=T1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">1</a>). Several sequences from all sample types were over 95&#x00025; similar to the <em>Gluconacetobacter diazotrophicus</em> sequence. Another group of eight sequences from North Inlet samples were substantially similar (&#x02265;95&#x00025;) to NifH sequences from rhizobia. Seven of these were over 97&#x00025; similar to a <em>Bradyrhizobium japonicum</em> sequence. Another substantial sequence grouping, predominantly from the Sapelo Island standing dead sample, was strongly affiliated with the NifH sequence from <em>Azospirillum brasilense</em>. Four of these had 99&#x00025; or greater similarity to the <em>A. brasilense</em> sequence, and two were identical to it. The NifH sequences of <em>A. brasilense</em> and <em>Azospirillum lipoferum</em> are 99.3&#x00025; similar, so at least four of the dead <em>Spartina</em> biomass NifH sequences were very likely from <em>Azospirillum</em> species and two seemingly from a strain of <em>A. brasilense</em>. This finding confirms the prediction of Newell et al. (<a href="#B19" class="cite-reflink">19</a>) that <em>Azospirillum</em>-like organisms may be involved in degradation of standing dead <em>Spartina</em> biomass. There were also many sequences that were not closely affiliated with any known diazotrophs among the &#x003b1;-<em>Proteobacteria</em>. Blast searches of the NCBI GenBank database revealed only a few sequences from other types of environmental samples that had meaningful similarity to any sequences from dead aboveground <em>Spartina</em> biomass. Among these were four sequences recovered from the <em>Spartina</em> rhizosphere (<a href="#B14" class="cite-reflink">14</a>).</p><p>The second major cluster containing sequences from dead aboveground <em>Spartina</em> biomass was characterized by sequences from &#x003b3;-<em>Proteobacteria</em> (Fig. <a href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">3</a>). This cluster contained three sequences from standing dead <em>Spartina</em> whose positions were ambiguous. When these three sequences were omitted and the tree was reconstructed, the &#x003b3;-<em>Proteobacteria</em> cluster was strongly supported by bootstrap analysis (60&#x00025; for the cluster as a whole, 96 and 91&#x00025; for the two major subclusters). Omitting these sequences also raised the bootstrap score for the &#x003b1;-<em>Proteobacteria</em> cluster from 76 to 99&#x00025;. One of the remaining seven presumptive &#x003b3;-<em>Proteobacteria</em> sequences was very strongly affiliated with the <em>Azotobacteriaceae</em>, with over 99&#x00025; similarity to the NifH sequences of <em>Azomonas agilis</em> and <em>Azotobacter chroococcum</em> (Table <a href="articlerender.fcgi?artid=93305&amp;rendertype=table&amp;id=T1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">1</a>). For comparison, the NifH sequences of <em>A. chroococcum</em> and <em>Azotobacter vinelandii</em> are 99.3&#x00025; similar. Two other sequences also had substantial similarity (&#x02265;95&#x00025;) to sequences from azotobacteria. The only other sequence with strong similarity to a sequence from a known organism was a North Inlet standing dead sequence that was &#x0003e;98&#x00025; similar to the sequence from <em>Pseudomonas stutzeri</em>. As was the case for sequences from presumptive &#x003b1;-<em>Proteobacteria</em>, few environmental sequences were substantially similar to those from dead aboveground <em>Spartina</em> biomass, and these were from the <em>Spartina</em> rhizosphere.  </p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F3" name="F3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F3" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=93305&amp;blobname=am1110925003.gif" border="1" class="icon-reflink" alt="FIG. 3" title="FIG. 3"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>FIG. 3</strong></a><div class="figure-table-caption-in-article"><span>Phylogenetic analysis of dead aboveground <em>Spartina</em> biomass NifH amino acid sequences from presumptive &#x003b3;-<em>Proteobacteria</em>, from various known &#x003b3;-<em>Proteobacteria</em>, and from selected unknown, presumptive &#x003b3;-<em>Proteobacteria</em> from other sources.</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F3" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>Only five sequences recovered from dead aboveground <em>Spartina</em> biomass were affiliated with the remaining major NifH sequence cluster, the anaerobes (Fig. <a href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F4" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">4</a>). While the closest affiliations of these sequences to any from known organisms were all with sulfate-reducing bacteria, none of the dead <em>Spartina</em> NifH sequences were sufficiently similar to any sequence from a known diazotroph to permit even presumptive identification. Two sequences from the North Inlet standing dead sample were similar to sequences from <em>Spartina</em> rhizosphere, but since the rhizosphere core samples contained both roots and sediments, and the dead aboveground biomass samples all carried a small amount of sediment, these sequences may be from sediment diazotrophs rather than species closely affiliated with the decomposing plant materials.  </p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F4" name="F4"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F4" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=93305&amp;blobname=am1110925004.gif" border="1" class="icon-reflink" alt="FIG. 4" title="FIG. 4"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F4" onclick="startTarget(this, 'figure', 1024, 800)"><strong>FIG. 4</strong></a><div class="figure-table-caption-in-article"><span>Phylogenetic analysis of dead aboveground <em>Spartina</em> biomass NifH amino acid sequences from unknown, presumptive anaerobic bacterial sequences, from various known anaerobic bacteria, and from selected unknown, presumptive anaerobic bacteria from other sources.</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=93305&amp;rendertype=figure&amp;id=F4" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>NifH sequences recovered from dead <em>Spartina</em> biomass reflected a diazotroph assemblage of very limited diversity, consisting almost exclusively of &#x003b1;-<em>Proteobacteria</em>. Some of these organisms are apparently related to known diazotrophs, including <em>Azospirillum brasilense</em>, <em>Bradyrhizobium japonicum</em>, <em>Gluconacetobacter diazotrophicus</em>, and <em>Rhodobacter sphaeroides</em>. Very few sequences from dead <em>Spartina</em> biomass were affiliated with the other major NifH sequence groups. However, it is noteworthy that among the few sequences recovered from presumptive &#x003b3;-<em>Proteobacteria</em>, several were quite similar to those from known diazotrophs, including <em>Azomonas agilis</em>, <em>Azotobacter chroococcum</em>, and <em>Pseudomonas stutzeri</em>. While it is certainly possible that some sequences were lost due to PCR biases or other artifacts (<a href="#B25" class="cite-reflink">25</a>, <a href="#B42" class="cite-reflink">42</a>), identical methods have yielded much more diverse sequence collections from other sample types (<a href="#B14" class="cite-reflink">14</a>; Lovell et al., unpublished data). It appears that while the relatively harsh (partially mineralized and frequently dry) microenvironments represented by dead aboveground <em>Spartina</em> biomass can support impressive rates of diazotrophy (<a href="#B19" class="cite-reflink">19</a>) when wet, they pose a substantial challenge for many diazotrophic biota and consequently harbor a very restricted range of organisms.</p></p><p><span class="head3">Nucleotide sequence accession numbers. </span> The <em>nifH</em> sequences determined in this study are available in the GenBank database under accession numbers <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0J5R1mj0-ZYumgDQNcrh2qIlDlYMz2-HDBIr-jAnG&amp;reftype=extlink-entrez-nucleotide&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CNucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=nucleotide&amp;dopt=GenBank&amp;list_uids=15186854" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">AF389702</a></span> to <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0bW1fXBFVXYL77HrR4FtX6DvisLwCLgZeKy59GQ0d&amp;reftype=extlink-entrez-nucleotide&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CNucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=nucleotide&amp;dopt=GenBank&amp;list_uids=15187096" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">AF389823</a></span>.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id671770">Acknowledgments</div><div class="section-content"><p><p>We acknowledge Steven Newell for assistance with Sapelo Island sampling site identification and the Belle W. Baruch Institute for Marine and Coastal Research for access to North Inlet sampling sites.</p><p>This research was supported by NSF award DEB-9903623 to C.R.L.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id701812" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>REFERENCES</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id671778">REFERENCES</div><div class="section-content"><div class="back-matter-section"><div class="ref-cit-blk" id="B1"><div class="ref-label">1.</div><div class="ref-cit">Affourtit, J; Zehr, J P; Paerl, H W. Distribution of nitrogen-fixing microorganisms along the Neuse River Estuary, North Carolina. <span><span class="ref-journal">Microb Ecol. </span>2001;<span class="ref-vol">41</span>:114–123.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=06yBXKBtJNYrdjFEfOuoHcftAvgPWWFoqbzbb8QVJ&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12032616">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B2"><div class="ref-label">2.</div><div class="ref-cit">Benson, D A; Boguski, M S; Lipman, D J; Ostell, J; Ouellette, B F F. GenBank. <span><span class="ref-journal">Nucleic Acids Res. </span>1998;<span class="ref-vol">26</span>:1–7.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0T5AygrAkE-MAru5edvYkPEDNZYADrhyjChIn5ktl&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9399790">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B3"><div class="ref-label">3.</div><div class="ref-cit">Braun, S T; Proctor, L M; Zani, S; Mellon, M T; Zehr, J P. Molecular evidence for zooplankton-associated nitrogen-fixing anaerobes based on amplification of the <em>nifH</em> gene. <span><span class="ref-journal">FEMS Microbiol Ecol. </span>1999;<span class="ref-vol">28</span>:273–279.</span></div></div><div class="ref-cit-blk" id="B4"><div class="ref-label">4.</div><div class="ref-cit">Dean, D R; Jacobson, M R.  Biochemical genetics of nitrogenase. In: Stacey G, Burris R H, Evans H J. ,  editors; Stacey G, Burris R H, Evans H J. ,  editors. <span class="ref-journal">Biological nitrogen fixation.</span> New York, N.Y: Chapman and Hall; 1992. pp. 763–834.</div></div><div class="ref-cit-blk" id="B5"><div class="ref-label">5.</div><div class="ref-cit">Devereux, J; Haeberli, P; Smithies, O. A comprehensive set of sequence analysis programs for the VAX. <span><span class="ref-journal">Nucleic Acids Res. </span>1984;<span class="ref-vol">12</span>:387–395.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Liv6cWeYzhWKSqDoczaVMjXWpo1_tyef-IFka5FW&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/6546423">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B6"><div class="ref-label">6.</div><div class="ref-cit">Felsenstein, J. Confidence limits on phylogenies: an approach using the bootstrap. <span><span class="ref-journal">Evolution. </span>1985;<span class="ref-vol">39</span>:783–791.</span></div></div><div class="ref-cit-blk" id="B7"><div class="ref-label">7.</div><div class="ref-cit">Gallagher, J L; Reimold, R J; Linthurst, R A; Pfeiffer, W J. Aerial production, mortality, and mineral accumulation-export dynamics in <em>Spartina alterniflora</em> and <em>Juncus roemerianus</em> plant stands in a Georgia salt marsh. <span><span class="ref-journal">Ecology. </span>1980;<span class="ref-vol">61</span>:303–312.</span></div></div><div class="ref-cit-blk" id="B8"><div class="ref-label">8.</div><div class="ref-cit">Hanson, R B. Nitrogen fixation activity (acetylene reduction) in the rhizosphere of salt marsh angiosperms, Georgia, USA. <span><span class="ref-journal">Bot Mar. </span>1983;<span class="ref-vol">26</span>:49–59.</span></div></div><div class="ref-cit-blk" id="B9"><div class="ref-label">9.</div><div class="ref-cit">Head, I M; Saunders, J R; Pickup, R W. Microbial evolution, diversity, and ecology: a decade of ribosomal RNA analysis of uncultivated microorganisms. <span><span class="ref-journal">Microb Ecol. </span>1998;<span class="ref-vol">35</span>:1–21.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Vw-A76fZfgDlTBurWzyKJ36r1SqpJTjtiOfcHkkF&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9459655">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B10"><div class="ref-label">10.</div><div class="ref-cit">Kirshtein, J D; Paerl, H W; Zehr, J. Amplification, cloning, and sequencing of a <em>nifH</em> segment from aquatic microorganisms and natural communities. <span><span class="ref-journal">Appl Environ Microbiol. </span>1991;<span class="ref-vol">57</span>:2645–2650.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0JeLgez9BPHOeATgIMQN4K6ZIQqiMgSKc3Qm39k53&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/1768139">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B11"><div class="ref-label">11.</div><div class="ref-cit">Kudo, T; Ohkuma, M; Moriya, S; Noda, S; Ohtoko, K. Molecular phylogenetic identification of the intestinal anaerobic microbial community in the hindgut of the termite Reticulitermes speratus, without cultivation. <span><span class="ref-journal">Extremophiles. </span>1998;<span class="ref-vol">2</span>:155–161.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0r1cfG2rreJNQfxtnOokKmrvgH3TxCwIuJt1lv56i&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9783160">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B12"><div class="ref-label">12.</div><div class="ref-cit"><strong>Kumar, S., K. Tamura, I. B. Jakobsen, and M. Nei.</strong> MEGA2: molecular evolutionary genetics analysis, version 2.0. Bioinformatics, in press.</div></div><div class="ref-cit-blk" id="B13"><div class="ref-label">13.</div><div class="ref-cit">Lovell, C R; Piceno, Y M. Purification of DNA from estuarine sediments. <span><span class="ref-journal">J Microbiol Methods. </span>1994;<span class="ref-vol">20</span>:161–174.</span></div></div><div class="ref-cit-blk" id="B14"><div class="ref-label">14.</div><div class="ref-cit">Lovell, C R; Piceno, Y M; Quattro, J M; Bagwell, C E. Molecular analysis of diazotroph diversity in the rhizosphere of the smooth cordgrass <em>Spartina alterniflora</em>. <span><span class="ref-journal">Appl Environ Microbiol. </span>2000;<span class="ref-vol">66</span>:3814–3822.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Ti3lpN0asdx4ggzH_03CZLbGH3gjpgRVuqM-jQyA&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10966395">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B15"><div class="ref-label">15.</div><div class="ref-cit">Marinucci, A C; Hobbie, J E; Helfrich, J V K. Effect of litter nitrogen on decomposition and microbial biomass in <em>Spartina alterniflora</em>. <span><span class="ref-journal">Microb Ecol. </span>1983;<span class="ref-vol">9</span>:27–40.</span></div></div><div class="ref-cit-blk" id="B16"><div class="ref-label">16.</div><div class="ref-cit">Morris, J T. Effects of nitrogen loading on wetland ecosystems with particular reference to atmospheric deposition. <span><span class="ref-journal">Annu Rev Ecol Syst. </span>1991;<span class="ref-vol">22</span>:257–279.</span></div></div><div class="ref-cit-blk" id="B17"><div class="ref-label">17.</div><div class="ref-cit">Newell, S Y; Fallon, R D; Cal Rodriguez, R M; Groene, L C. Influence of rain, tidal wetting and relative humidity on release of carbon dioxide by standing-dead salt-marsh plants. <span><span class="ref-journal">Oecologia. </span>1985;<span class="ref-vol">68</span>:73–79.</span></div></div><div class="ref-cit-blk" id="B18"><div class="ref-label">18.</div><div class="ref-cit">Newell, S Y; Fallon, R D; Miller, J D. Decomposition and microbial dynamics for standing, naturally positioned leaves of the salt-marsh grass <em>Spartina alterniflora</em>. <span><span class="ref-journal">Mar Biol. </span>1989;<span class="ref-vol">101</span>:471–481.</span></div></div><div class="ref-cit-blk" id="B19"><div class="ref-label">19.</div><div class="ref-cit">Newell, S Y; Hopkinson, C S; Scott, L A. Patterns of nitrogenase activity (acetylene reduction) associated with standing, decaying shoots of <em>Spartina alterniflora</em>. <span><span class="ref-journal">Estuar Coast Shelf Sci. </span>1992;<span class="ref-vol">35</span>:127–140.</span></div></div><div class="ref-cit-blk" id="B20"><div class="ref-label">20.</div><div class="ref-cit">Ohkuma, M; Noda, S; Usami, R; Horikoshi, K; Kudo, T. Diversity of nitrogen fixation genes in the symbiotic intestinal microflora of the termite <em>Reticulitermes speratus</em>. <span><span class="ref-journal">Appl Environ Microbiol. </span>1996;<span class="ref-vol">62</span>:2747–2752.</span></div></div><div class="ref-cit-blk" id="B21"><div class="ref-label">21.</div><div class="ref-cit">Olson, J B; Steppe, T F; Litaker, R W; Paerl, H W. N<sub>2</sub>-fixing microbial consortia associated with the ice cover of Lake Bonney, Antarctica. <span><span class="ref-journal">Microb Ecol. </span>1998;<span class="ref-vol">36</span>:231–238.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0XIWYuUp2jIQSoWtu8CejONCJtWELpIBfsJ5V3klg&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9852503">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B22"><div class="ref-label">22.</div><div class="ref-cit">Olson, J B; Litaker, R W; Paerl, H W. Ubiquity of heterotrophic diazotrophs in marine microbial mats. <span><span class="ref-journal">Aquat Microb Ecol. </span>1999;<span class="ref-vol">19</span>:29–36.</span></div></div><div class="ref-cit-blk" id="B23"><div class="ref-label">23.</div><div class="ref-cit">Patriquin, D G; McClung, C R. Nitrogen accretion, and the nature and possible significance of N<sub>2</sub> fixation (acetylene reduction) in a Nova Scotian <em>Spartina alterniflora</em> stand. <span><span class="ref-journal">Mar Biol. </span>1978;<span class="ref-vol">47</span>:227–242.</span></div></div><div class="ref-cit-blk" id="B24"><div class="ref-label">24.</div><div class="ref-cit">Peters, J W; Fisher, K; Dean, D R. Nitrogenase structure and function: a biochemical-genetic perspective. <span><span class="ref-journal">Annu Rev Microbiol. </span>1995;<span class="ref-vol">49</span>:335–366.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0xO0nWJbbQfIWPnlRayW72UNE5GDzjGS8j69-W57A&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8561464">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B25"><div class="ref-label">25.</div><div class="ref-cit">Piceno, Y M; Noble, P A; Lovell, C R. Spatial and temporal assessment of diazotroph assemblage composition in vegetated salt marsh sediments using denaturing gradient gel electrophoresis analysis. <span><span class="ref-journal">Microb Ecol. </span>1999;<span class="ref-vol">38</span>:157–167.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0B_pO-Pq7ih8GLsulX-njHExRv8a3AV_fyN6xHQb_&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10441708">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B26"><div class="ref-label">26.</div><div class="ref-cit">Piceno, Y M; Lovell, C R. Stability in natural microbial communities. I. Nutrient addition effects on rhizosphere diazotroph assemblage composition. <span><span class="ref-journal">Microb Ecol. </span>2000;<span class="ref-vol">39</span>:32–40.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0JC5utMmcCFAt_n-Dma18V00FumihUVazIY7-gQD2&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10790515">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B27"><div class="ref-label">27.</div><div class="ref-cit">Piceno, Y M; Lovell, C R. Stability in natural microbial communities. II. Plant resource allocation effects on rhizosphere diazotroph assemblage composition. <span><span class="ref-journal">Microb Ecol. </span>2000;<span class="ref-vol">39</span>:41–48.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0fSHnanxiX9H7NkM9an_91FDnNDJevxFPIduYs5sl&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10790516">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B28"><div class="ref-label">28.</div><div class="ref-cit">Poly, F; Ranjard, L; Nazaret, S; Gourbi&#x000e8;re, F; Monrozier, L J. Comparison of <em>nifH</em> gene pools in soils and soil microenvironments with contrasting properties. <span><span class="ref-journal">Appl Environ Microbiol. </span>2001;<span class="ref-vol">67</span>:2255–2262.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=04EOHn0wrSdUyQ4_52PSsPyNhQ0rWDhfk2IlxZ5H5&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11319109">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B29"><div class="ref-label">29.</div><div class="ref-cit">Rosado, A S; Duarte, G F; Seldin, L; Van Elsas, J D. Genetic diversity of <em>nifH</em> gene sequences in <em>Paenibacillus azotofixans</em> strains and soil samples analyzed by denaturing gradient gel electrophoresis of PCR-amplified gene fragments. <span><span class="ref-journal">Appl Environ Microbiol. </span>1998;<span class="ref-vol">64</span>:2770–2779.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0FqPGTNOq9-rgamNTlfVxbR1kfpLcylnRkDYztQSk&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9687429">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B30"><div class="ref-label">30.</div><div class="ref-cit">Saitou, N; Nei, M. The neighbor-joining method: a new method for reconstructing phylogenetic trees. <span><span class="ref-journal">Mol Biol Evol. </span>1987;<span class="ref-vol">4</span>:406–425.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=065znLDKsRy4_Nh3n3-bcGMYuhKc-QVOBkfd6XAaZ&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/3447015">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B31"><div class="ref-label">31.</div><div class="ref-cit">Schubauer, J P; Hopkinson, C S. Above- and belowground emergent macrophyte production and turnover in a coastal marsh ecosystem, Georgia. <span><span class="ref-journal">Limnol Oceanogr. </span>1984;<span class="ref-vol">29</span>:1052–1065.</span></div></div><div class="ref-cit-blk" id="B32"><div class="ref-label">32.</div><div class="ref-cit">Shaffer, B T; Widmer, F; Porteous, L A; Seidler, R J. Temporal and spatial distribution of the <em>nifH</em> gene of N<sub>2</sub>-fixing bacteria in forests and clearcuts in western Oregon. <span><span class="ref-journal">Microb Ecol. </span>2000;<span class="ref-vol">39</span>:12–21.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0RsnNlBeCcoEnY4gPHQoJnwJ8LvXQ4B1UXKfKwko5&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10790513">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B33"><div class="ref-label">33.</div><div class="ref-cit">Thomson, A D; Webb, K L. The effect of chronic oil pollution on salt-marsh nitrogen fixation (acetylene reduction). <span><span class="ref-journal">Estuaries. </span>1984;<span class="ref-vol">7</span>:2–11.</span></div></div><div class="ref-cit-blk" id="B34"><div class="ref-label">34.</div><div class="ref-cit">Tunlid, A. Molecular biology: a linkage between microbial ecology, general ecology and organismal biology. <span><span class="ref-journal">Oikos. </span>1999;<span class="ref-vol">85</span>:177–189.</span></div></div><div class="ref-cit-blk" id="B35"><div class="ref-label">35.</div><div class="ref-cit">Turner, R E. Geographic variations in salt marsh macrophyte production: a review. <span><span class="ref-journal">Contrib Mar Sci. </span>1976;<span class="ref-vol">20</span>:47–68.</span></div></div><div class="ref-cit-blk" id="B36"><div class="ref-label">36.</div><div class="ref-cit">Ueda, T; Suga, Y; Yahiro, N; Matsuguchi, T. Remarkable N<sub>2</sub>-fixing bacterial diversity detected in rice roots by molecular evolutionary analysis of <em>nifH</em> gene sequences. <span><span class="ref-journal">J Bacteriol. </span>1995;<span class="ref-vol">177</span>:1414–1417.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Xa5NvZJ4AaXkWPT-F8bBtcy6TYCV2fjVJiGi35Rv&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7868622">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B37"><div class="ref-label">37.</div><div class="ref-cit">Valiela, I; Teal, J M.  Nutrient limitation in salt marsh vegetation. In: Riemold R J, Queen W H. ,  editors; Riemold R J, Queen W H. ,  editors. <span class="ref-journal">Ecology of halophytes.</span> New York, N.Y: Academic Press; 1974. pp. 547–563.</div></div><div class="ref-cit-blk" id="B38"><div class="ref-label">38.</div><div class="ref-cit">Valiela, I; Teal, J M; Allen, S D; Van Etten, R; Goehringer, D; Volkmann, S. Decomposition in salt marsh ecosystems: the phases and major factors affecting disappearance of above-ground organic matter. <span><span class="ref-journal">J Exp Mar Biol Ecol. </span>1985;<span class="ref-vol">89</span>:29–54.</span></div></div><div class="ref-cit-blk" id="B39"><div class="ref-label">39.</div><div class="ref-cit">Ward, D M; Ferris, M J; Nold, S C; Bateson, M M. A natural view of microbial biodiversity within hot spring cyanobacterial mat communities. <span><span class="ref-journal">Microbiol Mol Biol Rev. </span>1998;<span class="ref-vol">62</span>:1353–1370.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=061Q3fXQ504kIwg8DgwZcOxt_pL2PmYgFuY4lQQ7n&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9841675">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B40"><div class="ref-label">40.</div><div class="ref-cit">Whiting, G J; Morris, J T. Nitrogen fixation (C<sub>2</sub>H<sub>2</sub> reduction) in a salt marsh: its relationship to temperature and an evaluation of an <em>in situ</em> chamber technique. <span><span class="ref-journal">Soil Biol Biochem. </span>1986;<span class="ref-vol">18</span>:515–521.</span></div></div><div class="ref-cit-blk" id="B41"><div class="ref-label">41.</div><div class="ref-cit">Widmer, S; Shaffer, B T; Porteous, L A; Seidler, R J. Analysis of <em>nifH</em> gene pool complexity in soil and litter at a Douglas fir forest site in the Oregon Cascade mountain range. <span><span class="ref-journal">Appl Environ Microbiol. </span>1999;<span class="ref-vol">65</span>:374–380.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0pyhyJER9OYLHQgVvzQpgvMZ66vU5sT7RrsXyRk70&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9925556">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B42"><div class="ref-label">42.</div><div class="ref-cit">Zehr, J P; Capone, D G. Problems and promises of assaying the genetic potential for nitrogen fixation in the marine environment. <span><span class="ref-journal">Microb Ecol. </span>1996;<span class="ref-vol">32</span>:263–281.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0j_mtXUTl6zVeRUxrzDVRnk0FPgVIdJbXGoBzWkRN&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8849422">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B43"><div class="ref-label">43.</div><div class="ref-cit">Zehr, J P; Mellon, M; Braun, S; Litaker, W; Steppe, T; Paerl, H W. Diversity of heterotrophic nitrogen fixation genes in a marine cyanobacterial mat. <span><span class="ref-journal">Appl Environ Microbiol. </span>1995;<span class="ref-vol">61</span>:2527–2532.</span></div></div><div class="ref-cit-blk" id="B44"><div class="ref-label">44.</div><div class="ref-cit">Zehr, J P; Mellon, M T; Zani, S. New nitrogen-fixing microorganisms detected in oligotrophic oceans by amplification of nitrogenase (<em>nifH</em>) genes. <span><span class="ref-journal">Appl Environ Microbiol. </span>1998;<span class="ref-vol">64</span>:3444–3450.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0nW3S6-KNHlZZcAQYczZ2WXRH_gfAGQ6a_4CrGAGm&amp;reftype=pubmed&amp;artid=93305&amp;article-id=93305&amp;iid=2543&amp;issue-id=2543&amp;jid=83&amp;journal-id=83&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9726895">PubMed</a>]</span></div></div></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">Applied and Environmental Microbiology</span> are provided here courtesy of <br><b>American Society for Microbiology (ASM)</b></div>
      <div class="footer-link" id="fp-footer">
        <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
           | 
          <a class="link" href="./">PMC Home</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov">NCBI</a>
           | 
          <a class="link" href="http://www.nlm.nih.gov">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov">NIH</a>
           | 
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="about/privacypolicy.html">Privacy Policy</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
           | 
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></table></body></html><!--host:ipmc1-->

