<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nucleic Acids Research"><meta name="citation_title" content="Data mining tools for the Saccharomyces cerevisiae morphological database"><meta name="citation_authors" content="Taro L. Saito, Jun Sese, Yoichiro Nakatani, Fumi Sano, Masashi Yukawa, Yoshikazu Ohya, and  Shinichi Morishita"><meta name="citation_date" content="2005 July 1"><meta name="citation_issue" content="Web Server issue"><meta name="citation_volume" content="33"><meta name="citation_firstpage" content="W753"><meta name="citation_doi" content="10.1093/nar/gki451"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=1160212&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1160212&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1160212"><meta name="citation_pmid" content="15980577"><meta name="DC.Title" content="Data mining tools for the Saccharomyces cerevisiae morphological database"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Taro L. Saito"><meta name="DC.Contributor" content="Jun Sese"><meta name="DC.Contributor" content="Yoichiro Nakatani"><meta name="DC.Contributor" content="Fumi Sano"><meta name="DC.Contributor" content="Masashi Yukawa"><meta name="DC.Contributor" content="Yoshikazu Ohya"><meta name="DC.Contributor" content="Shinichi Morishita"><meta name="DC.Date" content="2005 July 1"><meta name="DC.Identifier" content="10.1093/nar/gki451"><meta name="DC.Language" content="en"><link rel="stylesheet" href="corehtml/pmc/css/pmcstatic.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbase1.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbars-slateblue.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbody5.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcrefs2.css" type="text/css"><script type="text/javascript" src="corehtml/pmc/js/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/css/tileshop_pmc.1/tileshop_pmc.1.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/utils.1.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/tileshop_pmc.1/tileshop_pmc.1.js"></script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corecgi/tileshop/tileshop_data_db.1.js"></script><title>Data mining tools for the Saccharomyces cerevisiae morphological database</title></head><body class="pmc-body"><a id="top" name="top"></a><table cellpadding="0" cellspacing="0" width="100%" border="0"><tr valign="top"><td colspan="3"><table border="0" cellspacing="0" cellpadding="0" style="border-collapse: collapse;"><tr><td width="145" valign="top"><img src="corehtml/pmc/pmcgifs/pmc3_logo_v5.gif" border="0" usemap="#pmclogo" alt="pmc logo image"><map name="pmclogo"><area shape="rect" alt="Journal List" coords="65,52,137,67" href="fprender.fcgi"><area shape="rect" alt="Search" coords="7,52,52,67" href="redirect3.cgi?&amp;&amp;auth=0JAju2jzqSXy-pvvdPp5tWG9I1rLeqwyfTIkPDA09&amp;reftype=other&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="./"></map></td><td height="75" valign="top" style="padding-left: 4px;"><div><img align="top" border="0" src="corehtml/pmc/pmcgifs/narlogo.gif" alt="Logo of nar" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Journal URL: redirect3.cgi?&amp;&amp;auth=0Fe6cS9RagH0H3_-Xg1CNg1sZCIxedJr2elHHwkyU&amp;reftype=publisher&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article|Banner&amp;TO=Publisher|Other|N%2FA&amp;rendering-type=normal&amp;&amp;http://nar.oupjournals.org" shape="rect" coords="0,0,500,100" href="redirect3.cgi?&amp;&amp;auth=0pUz2_4qCGB1mrCAdy2Dc_IQ-He4NZy4x1NCaI5b2&amp;reftype=publisher&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://nar.oupjournals.org" onclick="focuswin('pmc_ext')" target="pmc_ext"></map></div></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="3" width="100%" border="0"><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-gray"><a href="fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?journal=4&amp;action=archive">Nucleic Acids Res</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?iid=119861">v.33(Web Server issue); Jul 1, 2005</a></div></td></tr><tr><td width="145" valign="top" class="sidebar-cell"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="articlerender.fcgi?artid=1160212&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="picrender.fcgi?artid=1160212&amp;blobtype=pdf" class="sidefm-pmclink">PDF (359K)</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?iid=119861" class="sidefm-pmclink">Contents</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?journal=4&amp;action=archive" class="sidefm-pmclink">Archive</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=0RYFWVzunqZMaA2gC0ej4mx4uI99fsIG9W-7fZksL&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15980577">PubMed record</option><option selected value="redirect3.cgi?&amp;&amp;auth=04gbLkNYugstsF6RqTG_7Ep_z07epkOc6hU5RPkTQ&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=15980577">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0VamZyJIvXH2JFKyFyOrOeF0qE8jZZjueFUqNL5zq&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=15980577">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=0Lay182dCPlQ3vcqkGx5lUQstCEqtaGqwnFvWGAIx&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1160212&amp;dopt=pmc_taxonomy">Taxonomy</option><option value="redirect3.cgi?&amp;&amp;auth=0Zy0g0-wIVtVuhr19NpQTPbvY-dDIOb0fUdaGn5LO&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?Db=taxonomy&amp;DbFrom=pmc&amp;cmd=Link&amp;LinkName=pmc_taxonomy&amp;doptcmdl=TxTree&amp;IdsFromResult=1160212">Taxonomy tree</option></select><br><input type="image" src="corehtml/pmc/pmcgifs/go.gif" alt="Go" class="sidefm-pmart"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0pwxeLU3MGccIIE5iMKUvw_yhl9dt02j-kEBJQQsZ&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Saito+TL%5Bauth%5D" class="sidefm-pmclink"> Saito, T.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0fIrnzY7IOFLibLTmuiePcnRYEQayXomPmQjCvQAF&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Sese+J%5Bauth%5D" class="sidefm-pmclink"> Sese, J.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0rs_i3zx1M_mgJeaVv49CxynwlKU89Mn3dH_Yn5H0&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Nakatani+Y%5Bauth%5D" class="sidefm-pmclink"> Nakatani, Y.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0lMJRmqtCFZChNsecGYJpLLpS3mi1mKNvKPaGpA6V&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Morishita+S%5Bauth%5D" class="sidefm-pmclink"> Morishita, S.</a></div></div></div></td><td rowspan="105" width="20px" height="300" background="corehtml/pmc/pmcgifs/wm-nar.gif"><div style="width:20px"></div></td><td valign="top" class="content-cell"><div class="front-matter-section"><table cellspacing="0" cellpadding="0" width="100%"><tr style="vertical-align: top"><td><div class="fm-citation"><div><span class="citation-abbreviation">Nucleic Acids Res. </span><span class="citation-publication-date">2005 July 1; </span><span class="citation-volume">33</span><span class="citation-issue">(Web Server issue)</span><span class="citation-flpages">: W753–W757. </span></div><div><span class="fm-vol-iss-date">Published online 2005 June 27. </span><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1093/nar/gki451.</span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC1160212</div></td></tr></table><div class="fm-copyright"><a class="int-reflink" href="about/copyright.html">Copyright</a> &#x000a9; The Author 2005. Published by Oxford University Press. All rights reserved</div><div class="fm-title">Data mining tools for the <em>Saccharomyces cerevisiae</em> morphological database</div><div class="fm-author contrib-group">Taro L. Saito,<sup>1,</sup><sup>4</sup> Jun Sese,<sup>2</sup> Yoichiro Nakatani,<sup>2,</sup><sup>4</sup> Fumi Sano,<sup>3,</sup><sup>4</sup> Masashi Yukawa,<sup>3,</sup><sup>4</sup> Yoshikazu Ohya,<sup>3,</sup><sup>4</sup> and  Shinichi Morishita<sup>2,</sup><sup>4</sup><sup>*</sup><div class="fm-affl"><sup>1</sup>Department of Computer Science, Graduate School of Information Science and Technology, University of Tokyo, 7-3-1 Hongo, Bunkyo-ku, Tokyo 113-0033, Japan</div><div class="fm-affl"><sup>2</sup>Department of Computational Biology, Graduate School of Frontier Sciences, University of Tokyo, Building FSB-101, 5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan</div><div class="fm-affl"><sup>3</sup>Department of Integrated Biosciences, Graduate School of Frontier Sciences, University of Tokyo, Building FSB-101, 5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8562, Japan</div><div class="fm-affl"><sup>4</sup>Japan and Institute for Bioinformatics and Research and Development, Japan Science and Technology Corporation, Science Plaza, 5-3, Yonbancho, Chiyoda-ku, Tokyo 102-8666, Japan</div></div><div class="fm-footnote"></div><div class="fm-footnote" id="cor1"><sup>*</sup>To whom correspondence should be addressed. Tel: +81 4 7136 3985; Fax: +81 4 7136 3977; Email: <span class="e_id574319">moris/at/k.u-tokyo.ac.jp</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id574319", '<a class="ext-reflink" href="' + reverseAndReplaceString('pj.ca.oykot-u.k/ta/sirom:otliam', '/at/', '@') + '">' + reverseAndReplaceString('pj.ca.oykot-u.k/ta/sirom', '/at/','@') + '</a>')}catch(e){}
                                //--></script></div><div class="fm-footnote"><div class="p">The authors wish it to be known that, in their opinion, the first two authors should be regarded as joint First Authors</div></div><div class="fm-pubdate">Received February 14, 2005; Revised March 31, 2005; Accepted March 31, 2005.</div><div class="fm-copyright"><p>The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact <span class="e_id551880">journals.permissions/at/oupjournals.org</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id551880", '<a class="ext-reflink" href="' + reverseAndReplaceString('gro.slanruojpuo/ta/snoissimrep.slanruoj:otliam', '/at/', '@') + '">' + reverseAndReplaceString('gro.slanruojpuo/ta/snoissimrep.slanruoj', '/at/','@') + '</a>')}catch(e){}
                                //--></script></p></div><div class="links-box"><div class="fm-footnote"><img src="corehtml/pmc/pmcgifs/rt-arrow.gif" alt="Small right arrow pointing to:" style="vertical-align: middle;"> This article has been <a class="int-reflink" href="tocrender.fcgi?artid=1160212&amp;action=cited">cited by</a> other articles in PMC.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Abstract</a></div><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id551911" class="sidefm-pmclink" style="text-transform: uppercase;">MORPHOLOGICAL DATABASE</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id551991" class="sidefm-pmclink" style="text-transform: uppercase;">CELL IMAGE PROCESSING</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552064" class="sidefm-pmclink" style="text-transform: uppercase;">DATA MINING</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552139" class="sidefm-pmclink" style="text-transform: uppercase;">CUSTOMIZATION AND DATA AVAILABILITY</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552150" class="sidefm-pmclink" style="text-transform: uppercase;">UPDATES AND FUTURE DIRECTIONS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552169" class="sidefm-pmclink" style="text-transform: uppercase;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id551883">Abstract</div><div class="section-content"><!--article-meta-->For comprehensive understanding of precise morphological changes resulting from loss-of-function mutagenesis, a large collection of 1 899 247 cell images was assembled from 91 271 micrographs of 4782 budding yeast disruptants of non-lethal genes. All the cell images were processed computationally to measure ~500 morphological parameters in individual mutants. We have recently made this morphological quantitative data available to the public through the <em>Saccharomyces cerevisiae</em> Morphological Database (SCMD). Inspecting the significance of morphological discrepancies between the wild type and the mutants is expected to provide clues to uncover genes that are relevant to the biological processes producing a particular morphology. To facilitate such intensive data mining, a suite of new software tools for visualizing parameter value distributions was developed to present mutants with significant changes in easily understandable forms. In addition, for a given group of mutants associated with a particular function, the system automatically identifies a combination of multiple morphological parameters that discriminates a mutant group from others significantly, thereby characterizing the function effectively. These data mining functions are available through the World Wide Web at <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0RGzKe52kxV2ydOQohNYLTrETkWpeYsgnRQeQ6kiT&amp;reftype=extlink&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://scmd.gi.k.u-tokyo.ac.jp/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://scmd.gi.k.u-tokyo.ac.jp/</a></span>.</div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id551883" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>MORPHOLOGICAL DATABASE</a></div><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id551991" class="sidefm-pmclink" style="text-transform: uppercase;">CELL IMAGE PROCESSING</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552064" class="sidefm-pmclink" style="text-transform: uppercase;">DATA MINING</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552139" class="sidefm-pmclink" style="text-transform: uppercase;">CUSTOMIZATION AND DATA AVAILABILITY</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552150" class="sidefm-pmclink" style="text-transform: uppercase;">UPDATES AND FUTURE DIRECTIONS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552169" class="sidefm-pmclink" style="text-transform: uppercase;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id551911">MORPHOLOGICAL DATABASE</div><div class="section-content"><p>To study the global regulation of cell morphological characteristics, a number of groups have recently reported genome-wide screening data for yeast mutants with abnormal morphology (<a href="#b1" class="cite-reflink">1</a>&#x02013;<a href="#b5" class="cite-reflink">5</a>). Despite the relatively simple ellipsoidal shape of yeast cells, in the past, cell morphology researchers processed information on cells manually. These time consuming, entirely subjective tasks motivated us to develop image-processing software called CalMorph (<a href="#b6" class="cite-reflink">6</a>), which automatically extracts yeast cells from micrographs and processes them to measure morphological characteristics such as cell size, roundness, bud neck position angle, nuclear position and actin localization. Using our software, we have retrieved 1 899 247 cells from 91 271 micrographs of 4782 mutants, which cover almost all of the yeast non-essential mutants cultured from the deleted strains available from EUROSCARF. All cell images, micrographs and quantitative values of morphological parameters are freely available from the SCMD database (<a href="#b7" class="cite-reflink">7</a>), which presents information that is complementary to the existing sequence and gene-expression databases (<a href="#b8" class="cite-reflink">8</a>&#x02013;<a href="#b12" class="cite-reflink">12</a>).</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id551883" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id551911" class="sidefm-pmclink" style="text-transform: uppercase;">MORPHOLOGICAL DATABASE</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>CELL IMAGE PROCESSING</a></div><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552064" class="sidefm-pmclink" style="text-transform: uppercase;">DATA MINING</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552139" class="sidefm-pmclink" style="text-transform: uppercase;">CUSTOMIZATION AND DATA AVAILABILITY</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552150" class="sidefm-pmclink" style="text-transform: uppercase;">UPDATES AND FUTURE DIRECTIONS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552169" class="sidefm-pmclink" style="text-transform: uppercase;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id551991">CELL IMAGE PROCESSING</div><div class="section-content"><p>Our software processes micrographs of cells stained with fluorescein isothiocyanate&#x02013;Concanavalin A (FITC-ConA) for cell wall identification, with DAPI to localize nuclei and with Rh-ph to visualize the actin distribution. The photos in <a href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1A</a> show three images stained with the respective dyes. <a href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1B</a> presents the result of combining three photos by superimposing images of the cell wall, nuclei and actin for individual cells.</p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig1" name="fig1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1160212&amp;blobname=gki451f1.gif" border="1" class="icon-reflink" alt="Figure 1" title="Figure 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 1</strong></a><div class="figure-table-caption-in-article"><span>Workflow of image processing and data mining. (<strong>A</strong>) Input photos of cells strained with FITC&#x02013;ConA, DAPI and Rh-ph to visualize the cell wall, nuclei and actin distribution, respectively. (<strong>B</strong>) Superimposition of three micrographs for individual cells.</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig1" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p><a href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1C</a> displays image-processing results. Our image-processing software first identifies the cell wall, attempts to fit an ellipse to each mother cell or bud and colors the cell wall green. The yellow lines show the long and short axes of the fitted ellipses. Bud necks that separate mother cells and buds are illustrated by using two red bullets. Identifying the cell wall makes it easier to determine information on the localization of nuclei and actin patches relative to the cell wall. In <a href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1C</a>, nuclei and actin patches are represented using yellow and light blue bullets, respectively.</p><p><a href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1D</a> shows the primary morphological parameters of cells. The quantitative values of these parameters may change slightly from cell to cell. To perform rigorous statistical analysis of the significance of morphological changes, we need to know the distribution of morphological parameter values for individual cells; this requires that we collect an ample number of image-processed cells and their parameter values. More than 200 image-processed cells were collected for each mutant using a sufficient number of micrographs. Then, ~500 morphological parameters were calculated for the mutants.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id551883" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id551911" class="sidefm-pmclink" style="text-transform: uppercase;">MORPHOLOGICAL DATABASE</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id551991" class="sidefm-pmclink" style="text-transform: uppercase;">CELL IMAGE PROCESSING</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>DATA MINING</a></div><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552139" class="sidefm-pmclink" style="text-transform: uppercase;">CUSTOMIZATION AND DATA AVAILABILITY</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552150" class="sidefm-pmclink" style="text-transform: uppercase;">UPDATES AND FUTURE DIRECTIONS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552169" class="sidefm-pmclink" style="text-transform: uppercase;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id552064">DATA MINING</div><div class="section-content"><p>Since there are so many parameters and mutants, some tools for assisting with data mining tasks will help users.</p><p><div class="head2 head-separate">Search</div>Morphological data should be useful for identifying the morphological changes in particular mutants. Users can query a yeast mutant of interest using its open reading frame name or its gene name. They can also browse average shapes of the mutant, average morphological parameter values, raw and processed micrographs and lists of individual cells associated with morphological parameter values. Users can also provide a typical morphological shape or a particular mutant as a query and ask the system to search for mutants that are similar in shape to the query. This function is called &#x02018;morphology search&#x02019; (<a href="#b7" class="cite-reflink">7</a>).</p><p><div class="head2 head-separate">Teardrop view&#x02014;juxtaposition of morphological parameter distributions</div>In order to understand which morphological parameters of a particular mutant are abnormal, the system displays the distribution of all mutants for each parameter and highlights the focal mutant value in pink (see <a href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2</a>). The system juxtaposes the distributions of all parameters in parallel, making it easy for users to comprehend the overview of distributions and abnormal parameters at a glance. Parameters are colored blue or pink if their changes are statistically significant in terms of their distributions.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig2" name="fig2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1160212&amp;blobname=gki451f2.gif" border="1" class="icon-reflink" alt="Figure 2" title="Figure 2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 2</strong></a><div class="figure-table-caption-in-article"><span>Teardrop view juxtaposes the morphological parameter distributions of all parameters for all mutants and the wild-type <em>HIS3</em> (<em>YOR202w</em>). For each morphological parameter, the distribution of all mutants and that of the wild type are displayed back-to-back</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig2" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><div class="head2 head-separate">Mutant classification in terms of morphological parameters</div>Another promising application of morphological parameters is to use them to predict gene functions. For instance, suppose that one is interested in finding a group of genes involved in a particular biological process such as DNA repairs and cell wall construction. You can ask the system to look for a combination of multiple morphological parameters that discriminate disruptants of genes that are known to be relevant to the biological process of interest (see <a href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 3</a>). These morphological parameters allow us to define distances between disruptants. If we identify disruptants that are not known to be related to any particular biological process but are closer to disruptants that are relevant to the focal biological process, these disrupted genes are potentially involved in the biological process.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig3" name="fig3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig3" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1160212&amp;blobname=gki451f3.gif" border="1" class="icon-reflink" alt="Figure 3" title="Figure 3"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 3</strong></a><div class="figure-table-caption-in-article"><span>Mutant classification in terms of morphological parameters. (<strong>A</strong>) Select a group of mutants such that the disrupted genes are involved in a biological process of interest. In the example, <em>CAP1</em> (<em>YKL007w</em>) and <em>CAP2</em> (<em>YIL034c</em>), capping protein and its beta subunit,</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1160212&amp;rendertype=figure&amp;id=fig3" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id552139">CUSTOMIZATION AND DATA AVAILABILITY</div><div class="section-content"><p>To facilitate customization according to users' interests for the ease of browsing, a dialog-based interface for the parameter selection page helps users choose parameters displayed in datasheets and are memorized in the system. The system also allows users to download the list of selected parameter values for selected mutants in the XML format or in tabular form. Users can also select particular mutants of interest so that they are always shown in Teardrop View and 2D plot.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id552150">UPDATES AND FUTURE DIRECTIONS</div><div class="section-content"><p>The web server currently presents morphological parameter values of disruptants of non-essential genes, but mutants of lethal genes will be processed and available in the future.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id552159">Acknowledgments</div><div class="section-content"><p><p>Funding to pay the Open Access publication charges for this article was provided by Japan Science and Technology Corporation.</p><p><em>Conflict of interest statement</em>. None declared.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id551883" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id551911" class="sidefm-pmclink" style="text-transform: uppercase;">MORPHOLOGICAL DATABASE</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id551991" class="sidefm-pmclink" style="text-transform: uppercase;">CELL IMAGE PROCESSING</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552064" class="sidefm-pmclink" style="text-transform: uppercase;">DATA MINING</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552139" class="sidefm-pmclink" style="text-transform: uppercase;">CUSTOMIZATION AND DATA AVAILABILITY</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id552150" class="sidefm-pmclink" style="text-transform: uppercase;">UPDATES AND FUTURE DIRECTIONS</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>REFERENCES</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id552169">REFERENCES</div><div class="section-content"><div class="back-matter-section"><div class="ref-cit-blk" id="b1"><div class="ref-label">1.</div><div class="ref-cit">Winzeler, E.A.; Shoemaker, D.D.; Astromoff, A.; Liang, H.; Anderson, K.; Andre, B.; Bangham, R.; Benito, R.; Boeke, J.D.; Bussey, H., et al.  Functional characterization of the <em>S.cerevisiae</em> genome by gene deletion and parallel analysis. <span><span class="ref-journal">Science. </span>1999;<span class="ref-vol">285</span>:901–906.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0psrMeW91VPD1dvIGux7cvLkOL2kmNimk06BZtQwf&amp;reftype=pubmed&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10436161">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="b2"><div class="ref-label">2.</div><div class="ref-cit">Jorgensen, P.; Nishikawa, J.L.; Breitkreutz, B.J.; Tyers, M. Systematic identification of pathways that couple cell growth and division in yeast. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">297</span>:395–400.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0hO0_tik3OKPSex7eJSCdtyFxVqN_-jblbGerLkw1&amp;reftype=pubmed&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12089449">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="b3"><div class="ref-label">3.</div><div class="ref-cit">Zhang, J.; Schneider, C.; Ottmers, L.; Rodriguez, R.; Day, A.; Markwardt, J.; Schneider, B.L. Genomic scale mutant hunt identifies cell size homeostasis genes in <em>S.cerevisiae</em>. <span><span class="ref-journal">Curr. Biol. </span>2002;<span class="ref-vol">12</span>:1992–2001.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0BwYq03ojBgbGEocXvHpaCh8nJ17exL_0KY4aZ5sN&amp;reftype=pubmed&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12477387">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="b4"><div class="ref-label">4.</div><div class="ref-cit">Ni, L.; Snyder, M. A genomic study of the bipolar bud site selection pattern in <em>Saccharomyces cerevisiae</em>. <span><span class="ref-journal">Mol. Biol. Cell. </span>2001;<span class="ref-vol">12</span>:2147–2170.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0t1PT1-B5ruwYgmEccy92VPSHgrIVjklcXdd91APS&amp;reftype=pubmed&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11452010">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="b5"><div class="ref-label">5.</div><div class="ref-cit">Giaever, G.; Chu, A.M.; Ni, L.; Connelly, C.; Riles, L.; Veronneau, S.; Dow, S.; Lucau-Danila, A.; Anderson, K.; Andre, B., et al.  Functional profiling of the <em>Saccharomyces cerevisiae</em> genome. <span><span class="ref-journal">Nature. </span>2002;<span class="ref-vol">418</span>:387–391.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0ncp69ytUliyY8mApICGEeOcgVmuvIF1s08gPRQVE&amp;reftype=pubmed&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12140549">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="b6"><div class="ref-label">6.</div><div class="ref-cit">Ohtani, M.; Saka, A.; Sano, F.; Ohya, Y.; Morishita, S. Development of image processing program for yeast cell morphology. <span><span class="ref-journal">J. Bioinform. Comput. Biol. </span>2004;<span class="ref-vol">1</span>:695–709.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=06mBJAm3B1B-CTyp0GB35pjgQO-JjIUbtyIwmhk4d&amp;reftype=pubmed&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15290760">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="b7"><div class="ref-label">7.</div><div class="ref-cit">Saito, L.T.; Ohtani, M.; Sawai, H.; Sano, F.; Saka, A.; Watanabe, D.; Yukawa, M.; Ohya, Y.; Morishita, S. SCMD: <em>Saccharomyces Cerevisiae</em> Morphological Database. <span><span class="ref-journal">Nucleic Acids Res. </span>2004;<span class="ref-vol">32</span>:319–322.</span></div></div><div class="ref-cit-blk" id="b8"><div class="ref-label">8.</div><div class="ref-cit">Balakrishnan, R.; Christie, K.; Costanzo, M.; Dolinski, K.; Dwight, S.; Engel, S.; Fisk, D.; Hirschman, J.; Hong, E.; Nash, R., et al.  Fungal BLAST and Model Organism BLASTP Best Hits: new comparison resources at the <em>Saccharomyces</em> Genome Database (SGD). <span><span class="ref-journal">Nucleic Acids Res. </span>2005;<span class="ref-vol">33</span>:D374–D377.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0bkj6HQTX50uYf0D7wLvx1xNS97F0FBAJaZXOcANH&amp;reftype=pubmed&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15608219">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="b9"><div class="ref-label">9.</div><div class="ref-cit">G&#x000fc;ldener, U.; M&#x000fc;nsterk&#x000f6;tter, M.; Kastenm&#x000fc;ller, G.; Strack, N.; van Helden, J.; Lemer, C.; Richelles, J.; Wodak, S.; Garc&#x000ed;a-Mart&#x000ed;nez, J.; P&#x000e9;rez-Ort&#x000ed;n, J., et al.  CYGD: the Comprehensive Yeast Genome Database. <span><span class="ref-journal">Nucleic Acids Res. </span>2005;<span class="ref-vol">33</span>:D364–D368.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0-a-0cOI_0ZEzGEV5uKWMVnsLrFPtOseBS8ioj51W&amp;reftype=pubmed&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15608217">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="b10"><div class="ref-label">10.</div><div class="ref-cit">Mewes, W.; Amid, C.; Arnold, R.; Frishman, D.; G&#x000fc;ldener, U.; Mannhaupt, G.; M&#x000fc;nsterk&#x000f6;tter, M.; Pagel, P.; Strack, N.; St&#x000fc;mpflen, V., et al.  MIPS: analysis and annotation of proteins from whole genomes. <span><span class="ref-journal">Nucleic Acids Res. </span>2004;<span class="ref-vol">32</span>:D41–D44.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0nGkWgvYGI1Pcf6o0g5iyqFgAPmk_aZXorELKX5ac&amp;reftype=pubmed&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/14681354">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="b11"><div class="ref-label">11.</div><div class="ref-cit">Riffle, M.; Malmstr&#x000f6;m, L.; Davis, T. The Yeast Resource Center Public Data Repository. <span><span class="ref-journal">Nucleic Acids Res. </span>2005;<span class="ref-vol">33</span>:D378–D382.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0fWgrmdVLjhIgr6SGEbWgwvCmFiEisEkaGbylW57l&amp;reftype=pubmed&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15608220">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="b12"><div class="ref-label">12.</div><div class="ref-cit">Lelandais, G.; Crom, S.; Devaux, F.; Vialette, S.; Church, G.; Jacq, C.; Marc, P. yMGV: a cross-species expression data mining tool. <span><span class="ref-journal">Nucleic Acids Res. </span>2004;<span class="ref-vol">32</span>:D323–D325.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=04GuuKjq_UM3cPY8_HDtXZPw8BAcOZYBOyTdKj5Zw&amp;reftype=pubmed&amp;artid=1160212&amp;article-id=1160212&amp;iid=119861&amp;issue-id=119861&amp;jid=4&amp;journal-id=4&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/14681424">PubMed</a>]</span></div></div></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">Nucleic Acids Research</span> are provided here courtesy of <br><b>Oxford University Press</b></div>
      <div class="footer-link" id="fp-footer">
        <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
           | 
          <a class="link" href="./">PMC Home</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov">NCBI</a>
           | 
          <a class="link" href="http://www.nlm.nih.gov">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov">NIH</a>
           | 
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="about/privacypolicy.html">Privacy Policy</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
           | 
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></table></body></html><!--host:ipmc3-->

